django_project/geocontext/utilities.py

```diff
@@ -101,7 +101,12 @@ def find_geometry_in_xml(url):
             try:
                 if 'gml' in sequence.attrib['type']:
                     geometry_name = sequence.attrib['name']
-                    geometry_type = sequence.attrib['type'].replace('gml:', '').replace('PropertyType', '')
+                    geometry_type = \
```

Comment:
please use parentheses for this case, it will look better :)

Reply:
Done